Oqtane.Server/Repository/Interfaces/IAliasRepository.cs

```diff
@@ -9,6 +9,7 @@ public interface IAliasRepository
         Alias AddAlias(Alias alias);
         Alias UpdateAlias(Alias alias);
         Alias GetAlias(int aliasId);
+        Alias GetAlias(string name);
```

Comment:
This is a breaking change!!

Reply:
The Repository is not considered part of the public API in Oqtane - only the actual Web API is the public API. Adding an additional method should be fine - in fact it is the expected way to enhance an application ( rather than modifying or deleting methods ). Or are you referring to the fact that this method is defined as a method overload?

Reply:
> The Repository is not considered part of the public API in Oqtane

Ok

> Or are you referring to the fact that this method is defined as a method overload?

Also it could be an extension method if it's possible

Reply:
I am not sure why it would need to be an extension method?

Reply:
No need while repository APIs not considered to be a breaking changes. BTW could could you please list all the breaking changes in Oqtane in `CONTRIB.md` so everyone will know what are the breaking changes in Oqtane before submit a PR